sdk/communication/communication-short-codes/swagger/README.md

```diff
@@ -0,0 +1,45 @@
+# Azure Communication Services Short Codes Module
+
+> see https://aka.ms/autorest
+
+## Configuration
+
+```yaml
+package-name: azure-communication-short-codes
+description: Short code acquiring and management client
+package-version: 1.0.0-beta.1
+generate-metadata: false
+license-header: MICROSOFT_MIT_NO_VERSION
+output-folder: ../src/generated
+tag: package-shortcode-2021-10-25-preview
+require: https://raw.githubusercontent.com/rperezzamora/azure-rest-api-specs/rperezzamora/addingNewContentCategories/specification/communication/data-plane/ShortCodes/readme.md
```

Comment:
Let's please reference the Azure owned public repo instead of a fork.

Reply:
Should I reference the latest or a specific commit?